src/components/toast/toast.spec.tsx

```diff
@@ -432,6 +436,42 @@ describe("Toast", () => {
     });
   });
 
+  describe("Correct icon is rendered", () => {
+    let wrapper;
+    it("should render `info` icon when variant is neutral", () => {
+      wrapper = mount(
+        <Toast open variant="neutral">
+          foo
+        </Toast>
+      );
+
+      expect(wrapper.find(TypeIcon).exists()).toBe(true);
+      expect(wrapper.find(Icon).prop("type")).toBe("info");
+    });
+
+    it("should render `tick_circle` icon when variant is success", () => {
+      wrapper = mount(
+        <Toast open variant="success">
+          foo
+        </Toast>
+      );
+
+      expect(wrapper.find(TypeIcon).exists()).toBe(true);
+      expect(wrapper.find(Icon).prop("type")).toBe("tick_circle");
+    });
+
+    it("should render `warning` icon when variant is warning", () => {
+      wrapper = mount(
+        <Toast open variant="warning">
+          foo
+        </Toast>
+      );
+
+      expect(wrapper.find(TypeIcon).exists()).toBe(true);
+      expect(wrapper.find(Icon).prop("type")).toBe("warning");
+    });
```

Comment:
**suggestion(non-blocking)**: what do you think about placing these tests into a parameterised block to simplify the code?


```suggestion
    it.each([
      ["info", "neutral"],
      ["tick_circle", "success"],
      ["warning", "warning"],
    ] as const)(`should render %s when variant is %s`, (icon, variant) => {
      const wrapper = mount(
        <Toast open variant={variant}>
          foo
        </Toast>
      );

      expect(wrapper.find(TypeIcon).exists()).toBe(true);
      expect(wrapper.find(Icon).prop("type")).toBe(icon);
    });
```